icu4c/source/common/rbbi.cpp

```diff
@@ -1196,7 +1205,7 @@ getLanguageBreakEngineFromFactory(UChar32 c)
     }
 
     int32_t i = gLanguageBreakFactories->size();
-    const LanguageBreakEngine *lbe = NULL;
+    LanguageBreakEngine *lbe = NULL;
```

Comment:
this is surely wrong, LanguageBreakEngine should be immutable and not changed once created, you need to keep it as a const pointer.

Reply:
Fixed